builtin/logical/database/rotation_test.go

```diff
@@ -1620,6 +1620,56 @@ func TestDeletesOlderWALsOnLoad(t *testing.T) {
 	requireWALs(t, storage, 1)
 }
 
+// TestStaticRoleNextVaultRotationOnPopulate tests the case where
+// a static role was created before the 1.15.0 release and the
+// NextVaultRotation was not set. It ensures that the NextVaultRotation
+// is set to the next rotation time when the role is read from storage
+// and the queue is populated.
+func TestStaticRoleNextVaultRotationOnPopulate(t *testing.T) {
```

Comment:
It would be great if you could format this as a table driven test. It would ensure that the tests are isolated from one another. It also makes it easier to read.